packages/text-field/helper-text/index.js

```diff
@@ -12,6 +12,7 @@ export default class HelperText extends React.Component {
     this.state = {
       'aria-hidden': props['aria-hidden'],
       'role': props.role,
+      'classList': new Set(),
```

Comment:
Are all these attributes are wrapped in single quotes to satisfy ESLint?

Reply:
correct...only because of aria-hidden

Reply:
Good to know, thanks